pkgs/development/libraries/nyan/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, clang_14
+, cmake
+, flex
+}:
+
+stdenv.mkDerivation {
+  pname = "nyan";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "SFTtech";
+    repo = "nyan";
+    rev = "7b5517f8c3c17b1fb2483f6dec882ccb84b70bbb";
```

Comment:
Risking being redundant, the upstream project is not providing tags? It is suboptimal to use raw Git revision hashes as addresses.

Reply:
I tried that and it kept breaking